charms/kubernetes_snaps.py

```diff
@@ -135,7 +135,7 @@ def configure_apiserver(
                 "Authorization mode includes 'Webhook' but no webhook config file is present."
                 "'Webhook' must be removed from authorization mode."
             )
-            authorization_modes = authorization_modes.remove("Webhook")
```

Comment:
This is the original thrust of the bug.  Python's list.remove returns None.  causing the following stack track:

```
  File "/var/lib/juju/agents/unit-kubernetes-control-plane-0/charm/./src/charm.py", line 541, in reconcile
    self.configure_apiserver()
  File "/var/lib/juju/agents/unit-kubernetes-control-plane-0/charm/./src/charm.py", line 141, in configure_apiserver
    kubernetes_snaps.configure_apiserver(
  File "/var/lib/juju/agents/unit-kubernetes-control-plane-0/charm/venv/charms/kubernetes_snaps.py", line 145, in configure_apiserver
    api_opts["authorization-mode"] = ",".join(authorization_modes)
TypeError: can only join an iterable
```